docs/migrate.rst

```diff
@@ -4,6 +4,10 @@ Migration Guide
 Migrating to Rally 2.7.1
 ------------------------
 
+Elasticsearch client logs are now captured by the ``elastic_transport`` logger
```

Comment:
Maybe link to the library's GitHub repo? Not a big deal, though.

Reply:
> This is really great work @b-deam.
> 
> One question/suggestion: Do you think it's worth running a CCS benchmark with this PR? A priori, I don't think there would be any issues there, but instantiating clients for multiple clusters is a code path that isn't otherwise exercised by the testing you've done, so it may be worth a sanity check.

Good suggestion. I tested the CCS challenge locally, but for completeness' sake I ran a comparison using the nightly configuration. The results look OK to me.

<details>
<summary> ccs </summary>

```
baseline: 218b1a1e-e73e-4aba-a895-bdfe6ac5d47c
contender: 51ed0fb6-5e2a-476f-9e86-00bcbdd0ec19
$ esrally compare --baseline 218b1a1e-e73e-4aba-a895-bdfe6ac5d47c --contender 51ed0fb6-5e2a-476f-9e86-00bcbdd0ec19

    ____        ____
   / __ \____ _/ / /_  __
  / /_/ / __ `/ / / / / /
 / _, _/ /_/ / / / /_/ /
/_/ |_|\__,_/_/_/\__, /
                /____/


Comparing baseline
  Race ID: 218b1a1e-e73e-4aba-a895-bdfe6ac5d47c
  Race timestamp: 2022-12-25 20:00:00
  Challenge: cross-clusters-search-and-snapshot
  Car: external
  User tags: benchmark-name=ccs, benchmark-type=rally-es-client-upgrade, created-by=esbench, division=engineering, env-id=27e4c9c7-e516-4572-8d56-9cba273853a7, git-username=Brad Deam, host-username=bradleydeam, org=elasticsearch, project=esbench-nightly, setup=esbench, team=performance

with contender
  Race ID: 51ed0fb6-5e2a-476f-9e86-00bcbdd0ec19
  Race timestamp: 2022-12-25 20:00:00
  Challenge: cross-clusters-search-and-snapshot
  Car: external
  User tags: benchmark-name=ccs, benchmark-type=rally-es-client-upgrade, created-by=esbench, division=engineering, env-id=e7ec9d18-e84f-46a3-9ea4-5d113ec9e579, git-username=Brad Deam, host-username=bradleydeam, org=elasticsearch, project=esbench-nightly, setup=esbench, team=performance

------------------------------------------------------
    _______             __   _____
   / ____(_)___  ____ _/ /  / ___/_________  ________
  / /_  / / __ \/ __ `/ /   \__ \/ ___/ __ \/ ___/ _ \
 / __/ / / / / / /_/ / /   ___/ / /__/ /_/ / /  /  __/
/_/   /_/_/ /_/\__,_/_/   /____/\___/\____/_/   \___/
------------------------------------------------------
            
|                                                        Metric |                                                                  Task |         Baseline |        Contender |          Diff |   Unit |   Diff % |
|--------------------------------------------------------------:|----------------------------------------------------------------------:|-----------------:|-----------------:|--------------:|-------:|---------:|
|                    Cumulative indexing time of primary shards |                                                                       |    121.781       |    120.467       |      -1.3146  |    min |   -1.08% |
|             Min cumulative indexing time across primary shard |                                                                       |      0.00378333  |      0.00381667  |       3e-05   |    min |   +0.88% |
|          Median cumulative indexing time across primary shard |                                                                       |      0.0186333   |      0.0182333   |      -0.0004  |    min |   -2.15% |
|             Max cumulative indexing time across primary shard |                                                                       |      0.25585     |      0.259267    |       0.00342 |    min |   +1.34% |
|           Cumulative indexing throttle time of primary shards |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|    Min cumulative indexing throttle time across primary shard |                                                                       |      0           |      0           |       0       |    min |    0.00% |
| Median cumulative indexing throttle time across primary shard |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|    Max cumulative indexing throttle time across primary shard |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|                       Cumulative merge time of primary shards |                                                                       |    121.869       |    125.728       |       3.85937 |    min |   +3.17% |
|                      Cumulative merge count of primary shards |                                                                       |  25085           |  25279           |     194       |        |   +0.77% |
|                Min cumulative merge time across primary shard |                                                                       |      0.000183333 |      0.0002      |       2e-05   |    min |   +9.09% |
|             Median cumulative merge time across primary shard |                                                                       |      0.0025      |      0.0027      |       0.0002  |    min |   +8.00% |
|                Max cumulative merge time across primary shard |                                                                       |      0.48085     |      0.528917    |       0.04807 |    min |  +10.00% |
|              Cumulative merge throttle time of primary shards |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|       Min cumulative merge throttle time across primary shard |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|    Median cumulative merge throttle time across primary shard |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|       Max cumulative merge throttle time across primary shard |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|                     Cumulative refresh time of primary shards |                                                                       |    109.815       |    110.781       |       0.96625 |    min |   +0.88% |
|                    Cumulative refresh count of primary shards |                                                                       | 201570           | 202507           |     937       |        |   +0.46% |
|              Min cumulative refresh time across primary shard |                                                                       |      0.00025     |      0.000283333 |       3e-05   |    min |  +13.33% |
|           Median cumulative refresh time across primary shard |                                                                       |      0.00705833  |      0.00736667  |       0.00031 |    min |   +4.37% |
|              Max cumulative refresh time across primary shard |                                                                       |      0.301733    |      0.308333    |       0.0066  |    min |   +2.19% |
|                       Cumulative flush time of primary shards |                                                                       |      2.23607     |      2.39573     |       0.15967 |    min |   +7.14% |
|                      Cumulative flush count of primary shards |                                                                       |   4756           |   4750           |      -6       |        |   -0.13% |
|                Min cumulative flush time across primary shard |                                                                       |      0           |      0           |       0       |    min |    0.00% |
|             Median cumulative flush time across primary shard |                                                                       |      0.000333333 |      0.0003      |      -3e-05   |    min |  -10.00% |
|                Max cumulative flush time across primary shard |                                                                       |      0.0120167   |      0.0116      |      -0.00042 |    min |   -3.47% |
|                                       Total Young Gen GC time |                                                                       |    151.959       |    151.062       |      -0.897   |      s |   -0.59% |
|                                      Total Young Gen GC count |                                                                       |   1555           |   1534           |     -21       |        |   -1.35% |
|                                         Total Old Gen GC time |                                                                       |      0           |      0           |       0       |      s |    0.00% |
|                                        Total Old Gen GC count |                                                                       |      0           |      0           |       0       |        |    0.00% |
|                                                    Store size |                                                                       |     10.6716      |     10.6759      |       0.00434 |     GB |   +0.04% |
|                                                 Translog size |                                                                       |      0.000166474 |      0.000166474 |       0       |     GB |    0.00% |
|                                        Heap used for segments |                                                                       |      0           |      0           |       0       |     MB |    0.00% |
|                                      Heap used for doc values |                                                                       |      0           |      0           |       0       |     MB |    0.00% |
|                                           Heap used for terms |                                                                       |      0           |      0           |       0       |     MB |    0.00% |
|                                           Heap used for norms |                                                                       |      0           |      0           |       0       |     MB |    0.00% |
|                                          Heap used for points |                                                                       |      0           |      0           |       0       |     MB |    0.00% |
|                                   Heap used for stored fields |                                                                       |      0           |      0           |       0       |     MB |    0.00% |
|                                                 Segment count |                                                                       |   3250           |   3250           |       0       |        |    0.00% |
|                                   Total Ingest Pipeline count |                                                                       |      9.83069e+07 |      9.83069e+07 |       0       |        |    0.00% |
|                                    Total Ingest Pipeline time |                                                                       |      1.19726e+07 |      1.14634e+07 | -509234       |     ms |   -4.25% |
|                                  Total Ingest Pipeline failed |                                                                       |      0           |      0           |       0       |        |    0.00% |
|                                                Min Throughput |                           delete-existing-datastreams-on-all-clusters |      6.79743     |      6.64217     |      -0.15526 |  ops/s |   -2.28% |
|                                               Mean Throughput |                           delete-existing-datastreams-on-all-clusters |      6.79743     |      6.64217     |      -0.15526 |  ops/s |   -2.28% |
|                                             Median Throughput |                           delete-existing-datastreams-on-all-clusters |      6.79743     |      6.64217     |      -0.15526 |  ops/s |   -2.28% |
|                                                Max Throughput |                           delete-existing-datastreams-on-all-clusters |      6.79743     |      6.64217     |      -0.15526 |  ops/s |   -2.28% |
|                                      100th percentile latency |                           delete-existing-datastreams-on-all-clusters |    292.499       |    299.258       |       6.75897 |     ms |   +2.31% |
|                                 100th percentile service time |                           delete-existing-datastreams-on-all-clusters |    292.499       |    299.258       |       6.75897 |     ms |   +2.31% |
|                                                    error rate |                           delete-existing-datastreams-on-all-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                tune-recovery-settings-on-all-clusters |      3.33942     |      3.37792     |       0.03849 |  ops/s |   +1.15% |
|                                               Mean Throughput |                                tune-recovery-settings-on-all-clusters |      3.33942     |      3.37792     |       0.03849 |  ops/s |   +1.15% |
|                                             Median Throughput |                                tune-recovery-settings-on-all-clusters |      3.33942     |      3.37792     |       0.03849 |  ops/s |   +1.15% |
|                                                Max Throughput |                                tune-recovery-settings-on-all-clusters |      3.33942     |      3.37792     |       0.03849 |  ops/s |   +1.15% |
|                                      100th percentile latency |                                tune-recovery-settings-on-all-clusters |    298.628       |    294.712       |      -3.91669 |     ms |   -1.31% |
|                                 100th percentile service time |                                tune-recovery-settings-on-all-clusters |    298.628       |    294.712       |      -3.91669 |     ms |   -1.31% |
|                                                    error rate |                                tune-recovery-settings-on-all-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                      insert-pipelines |     12.7278      |     11.5084      |      -1.2194  |  ops/s |   -9.58% |
|                                               Mean Throughput |                                                      insert-pipelines |     12.7278      |     11.5084      |      -1.2194  |  ops/s |   -9.58% |
|                                             Median Throughput |                                                      insert-pipelines |     12.7278      |     11.5084      |      -1.2194  |  ops/s |   -9.58% |
|                                                Max Throughput |                                                      insert-pipelines |     12.7278      |     11.5084      |      -1.2194  |  ops/s |   -9.58% |
|                                      100th percentile latency |                                                      insert-pipelines |   1177.02        |   1294           |     116.98    |     ms |   +9.94% |
|                                 100th percentile service time |                                                      insert-pipelines |   1177.02        |   1294           |     116.98    |     ms |   +9.94% |
|                                                    error rate |                                                      insert-pipelines |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                            insert-ilm |     27.9609      |     27.291       |      -0.66988 |  ops/s |   -2.40% |
|                                               Mean Throughput |                                                            insert-ilm |     27.9609      |     27.291       |      -0.66988 |  ops/s |   -2.40% |
|                                             Median Throughput |                                                            insert-ilm |     27.9609      |     27.291       |      -0.66988 |  ops/s |   -2.40% |
|                                                Max Throughput |                                                            insert-ilm |     27.9609      |     27.291       |      -0.66988 |  ops/s |   -2.40% |
|                                      100th percentile latency |                                                            insert-ilm |     34.6168      |     34.5742      |      -0.04253 |     ms |   -0.12% |
|                                 100th percentile service time |                                                            insert-ilm |     34.6168      |     34.5742      |      -0.04253 |     ms |   -0.12% |
|                                                    error rate |                                                            insert-ilm |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                validate-package-template-installation |     45.8875      |     50.0293      |       4.14174 |  ops/s |   +9.03% |
|                                               Mean Throughput |                                validate-package-template-installation |     45.8875      |     50.0293      |       4.14174 |  ops/s |   +9.03% |
|                                             Median Throughput |                                validate-package-template-installation |     45.8875      |     50.0293      |       4.14174 |  ops/s |   +9.03% |
|                                                Max Throughput |                                validate-package-template-installation |     45.8875      |     50.0293      |       4.14174 |  ops/s |   +9.03% |
|                                      100th percentile latency |                                validate-package-template-installation |     21.3813      |     19.5102      |      -1.87109 |     ms |   -8.75% |
|                                 100th percentile service time |                                validate-package-template-installation |     21.3813      |     19.5102      |      -1.87109 |     ms |   -8.75% |
|                                                    error rate |                                validate-package-template-installation |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                       update-custom-package-templates |     25.8823      |     24.4688      |      -1.41351 |  ops/s |   -5.46% |
|                                               Mean Throughput |                                       update-custom-package-templates |     25.8823      |     24.4688      |      -1.41351 |  ops/s |   -5.46% |
|                                             Median Throughput |                                       update-custom-package-templates |     25.8823      |     24.4688      |      -1.41351 |  ops/s |   -5.46% |
|                                                Max Throughput |                                       update-custom-package-templates |     25.8823      |     24.4688      |      -1.41351 |  ops/s |   -5.46% |
|                                      100th percentile latency |                                       update-custom-package-templates |    463.235       |    489.939       |      26.7045  |     ms |   +5.76% |
|                                 100th percentile service time |                                       update-custom-package-templates |    463.235       |    489.939       |      26.7045  |     ms |   +5.76% |
|                                                    error rate |                                       update-custom-package-templates |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                            bulk-index |   5974.36        |   1619.33        |   -4355.03    | docs/s |  -72.90% |
|                                               Mean Throughput |                                                            bulk-index |   9446.91        |   9512.2         |      65.2976  | docs/s |   +0.69% |
|                                             Median Throughput |                                                            bulk-index |   9326.87        |   9388.87        |      61.9944  | docs/s |   +0.66% |
|                                                Max Throughput |                                                            bulk-index |  10526.6         |  16442.7         |    5916.02    | docs/s |  +56.20% |
|                                       50th percentile latency |                                                            bulk-index |   3070.81        |   3040.4         |     -30.4023  |     ms |   -0.99% |
|                                       90th percentile latency |                                                            bulk-index |   4271.13        |   4281.41        |      10.2838  |     ms |   +0.24% |
|                                       99th percentile latency |                                                            bulk-index |   6073.46        |   6110.36        |      36.9023  |     ms |   +0.61% |
|                                     99.9th percentile latency |                                                            bulk-index |   7507.86        |   7603.37        |      95.5115  |     ms |   +1.27% |
|                                    99.99th percentile latency |                                                            bulk-index |   9021.62        |   8695.52        |    -326.096   |     ms |   -3.61% |
|                                      100th percentile latency |                                                            bulk-index |   9807.61        |   8939.99        |    -867.621   |     ms |   -8.85% |
|                                  50th percentile service time |                                                            bulk-index |   3070.75        |   3040.16        |     -30.5882  |     ms |   -1.00% |
|                                  90th percentile service time |                                                            bulk-index |   4270.78        |   4282.5         |      11.715   |     ms |   +0.27% |
|                                  99th percentile service time |                                                            bulk-index |   6076.07        |   6112.52        |      36.4469  |     ms |   +0.60% |
|                                99.9th percentile service time |                                                            bulk-index |   7507.86        |   7603.37        |      95.5115  |     ms |   +1.27% |
|                               99.99th percentile service time |                                                            bulk-index |   9021.62        |   8695.52        |    -326.096   |     ms |   -3.61% |
|                                 100th percentile service time |                                                            bulk-index |   9807.61        |   8939.99        |    -867.621   |     ms |   -8.85% |
|                                                    error rate |                                                            bulk-index |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                       register-snapshot-repository-on-remote-clusters |      0.627022    |      0.616449    |      -0.01057 |  ops/s |   -1.69% |
|                                               Mean Throughput |                       register-snapshot-repository-on-remote-clusters |      0.627022    |      0.616449    |      -0.01057 |  ops/s |   -1.69% |
|                                             Median Throughput |                       register-snapshot-repository-on-remote-clusters |      0.627022    |      0.616449    |      -0.01057 |  ops/s |   -1.69% |
|                                                Max Throughput |                       register-snapshot-repository-on-remote-clusters |      0.627022    |      0.616449    |      -0.01057 |  ops/s |   -1.69% |
|                                      100th percentile latency |                       register-snapshot-repository-on-remote-clusters |   1594.15        |   1619.12        |      24.9664  |     ms |   +1.57% |
|                                 100th percentile service time |                       register-snapshot-repository-on-remote-clusters |   1594.15        |   1619.12        |      24.9664  |     ms |   +1.57% |
|                                                    error rate |                       register-snapshot-repository-on-remote-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                            create-snapshot-many-clusters-ccs-snapshot |     15.395       |     12.2879      |      -3.10718 |  ops/s |  -20.18% |
|                                               Mean Throughput |                            create-snapshot-many-clusters-ccs-snapshot |     15.395       |     12.2879      |      -3.10718 |  ops/s |  -20.18% |
|                                             Median Throughput |                            create-snapshot-many-clusters-ccs-snapshot |     15.395       |     12.2879      |      -3.10718 |  ops/s |  -20.18% |
|                                                Max Throughput |                            create-snapshot-many-clusters-ccs-snapshot |     15.395       |     12.2879      |      -3.10718 |  ops/s |  -20.18% |
|                                      100th percentile latency |                            create-snapshot-many-clusters-ccs-snapshot |     64.5192      |     79.2015      |      14.6823  |     ms |  +22.76% |
|                                 100th percentile service time |                            create-snapshot-many-clusters-ccs-snapshot |     64.5192      |     79.2015      |      14.6823  |     ms |  +22.76% |
|                                                    error rate |                            create-snapshot-many-clusters-ccs-snapshot |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                          wait-for-snapshot-many-clusters-ccs-snapshot |      3.13262e+07 |      3.16819e+07 |  355640       | byte/s |   +1.14% |
|                                               Mean Throughput |                          wait-for-snapshot-many-clusters-ccs-snapshot |      3.13262e+07 |      3.16819e+07 |  355640       | byte/s |   +1.14% |
|                                             Median Throughput |                          wait-for-snapshot-many-clusters-ccs-snapshot |      3.13262e+07 |      3.16819e+07 |  355640       | byte/s |   +1.14% |
|                                                Max Throughput |                          wait-for-snapshot-many-clusters-ccs-snapshot |      3.13262e+07 |      3.16819e+07 |  355640       | byte/s |   +1.14% |
|                                      100th percentile latency |                          wait-for-snapshot-many-clusters-ccs-snapshot | 518136           | 509989           |   -8146.97    |     ms |   -1.57% |
|                                 100th percentile service time |                          wait-for-snapshot-many-clusters-ccs-snapshot | 518136           | 509989           |   -8146.97    |     ms |   -1.57% |
|                                                    error rate |                          wait-for-snapshot-many-clusters-ccs-snapshot |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |        recover-snapshot-many-clusters-ccs-snapshot-on-remote-clusters |      0.170329    |      0.171223    |       0.00089 |  ops/s |   +0.52% |
|                                               Mean Throughput |        recover-snapshot-many-clusters-ccs-snapshot-on-remote-clusters |      0.170329    |      0.171223    |       0.00089 |  ops/s |   +0.52% |
|                                             Median Throughput |        recover-snapshot-many-clusters-ccs-snapshot-on-remote-clusters |      0.170329    |      0.171223    |       0.00089 |  ops/s |   +0.52% |
|                                                Max Throughput |        recover-snapshot-many-clusters-ccs-snapshot-on-remote-clusters |      0.170329    |      0.171223    |       0.00089 |  ops/s |   +0.52% |
|                                      100th percentile latency |        recover-snapshot-many-clusters-ccs-snapshot-on-remote-clusters |   5869.71        |   5837.79        |     -31.9185  |     ms |   -0.54% |
|                                 100th percentile service time |        recover-snapshot-many-clusters-ccs-snapshot-on-remote-clusters |   5869.71        |   5837.79        |     -31.9185  |     ms |   -0.54% |
|                                                    error rate |        recover-snapshot-many-clusters-ccs-snapshot-on-remote-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput | wait-for-snapshot-recovery-many-clusters-ccs-snapshot-on-all-clusters |      0.000949647 |      0.000988958 |       4e-05   |  ops/s |   +4.14% |
|                                               Mean Throughput | wait-for-snapshot-recovery-many-clusters-ccs-snapshot-on-all-clusters |      0.000949647 |      0.000988958 |       4e-05   |  ops/s |   +4.14% |
|                                             Median Throughput | wait-for-snapshot-recovery-many-clusters-ccs-snapshot-on-all-clusters |      0.000949647 |      0.000988958 |       4e-05   |  ops/s |   +4.14% |
|                                                Max Throughput | wait-for-snapshot-recovery-many-clusters-ccs-snapshot-on-all-clusters |      0.000949647 |      0.000988958 |       4e-05   |  ops/s |   +4.14% |
|                                      100th percentile latency | wait-for-snapshot-recovery-many-clusters-ccs-snapshot-on-all-clusters |      1.05302e+06 |      1.01116e+06 |  -41858.2     |     ms |   -3.98% |
|                                 100th percentile service time | wait-for-snapshot-recovery-many-clusters-ccs-snapshot-on-all-clusters |      1.05302e+06 |      1.01116e+06 |  -41858.2     |     ms |   -3.98% |
|                                                    error rate | wait-for-snapshot-recovery-many-clusters-ccs-snapshot-on-all-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                     check-cluster-health-all-clusters |      3.8425      |      3.75721     |      -0.08529 |  ops/s |   -2.22% |
|                                               Mean Throughput |                                     check-cluster-health-all-clusters |      3.8425      |      3.75721     |      -0.08529 |  ops/s |   -2.22% |
|                                             Median Throughput |                                     check-cluster-health-all-clusters |      3.8425      |      3.75721     |      -0.08529 |  ops/s |   -2.22% |
|                                                Max Throughput |                                     check-cluster-health-all-clusters |      3.8425      |      3.75721     |      -0.08529 |  ops/s |   -2.22% |
|                                      100th percentile latency |                                     check-cluster-health-all-clusters |    259.281       |    263.498       |       4.21741 |     ms |   +1.63% |
|                                 100th percentile service time |                                     check-cluster-health-all-clusters |    259.281       |    263.498       |       4.21741 |     ms |   +1.63% |
|                                                    error rate |                                     check-cluster-health-all-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                  setup-bi-directional-remote-clusters |      0.435378    |      0.432788    |      -0.00259 |  ops/s |   -0.59% |
|                                               Mean Throughput |                                  setup-bi-directional-remote-clusters |      0.435378    |      0.432788    |      -0.00259 |  ops/s |   -0.59% |
|                                             Median Throughput |                                  setup-bi-directional-remote-clusters |      0.435378    |      0.432788    |      -0.00259 |  ops/s |   -0.59% |
|                                                Max Throughput |                                  setup-bi-directional-remote-clusters |      0.435378    |      0.432788    |      -0.00259 |  ops/s |   -0.59% |
|                                      100th percentile latency |                                  setup-bi-directional-remote-clusters |   2296.34        |   2310.01        |      13.6631  |     ms |   +0.59% |
|                                 100th percentile service time |                                  setup-bi-directional-remote-clusters |   2296.34        |   2310.01        |      13.6631  |     ms |   +0.59% |
|                                                    error rate |                                  setup-bi-directional-remote-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                               reset-recovery-settings-on-all-clusters |      3.37303     |      3.31428     |      -0.05876 |  ops/s |   -1.74% |
|                                               Mean Throughput |                               reset-recovery-settings-on-all-clusters |      3.37303     |      3.31428     |      -0.05876 |  ops/s |   -1.74% |
|                                             Median Throughput |                               reset-recovery-settings-on-all-clusters |      3.37303     |      3.31428     |      -0.05876 |  ops/s |   -1.74% |
|                                                Max Throughput |                               reset-recovery-settings-on-all-clusters |      3.37303     |      3.31428     |      -0.05876 |  ops/s |   -1.74% |
|                                      100th percentile latency |                               reset-recovery-settings-on-all-clusters |    295.802       |    300.963       |       5.16156 |     ms |   +1.74% |
|                                 100th percentile service time |                               reset-recovery-settings-on-all-clusters |    295.802       |    300.963       |       5.16156 |     ms |   +1.74% |
|                                                    error rate |                               reset-recovery-settings-on-all-clusters |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                       discover/search |      0.00228331  |      0.00228458  |       0       |  ops/s |   +0.06% |
|                                               Mean Throughput |                                                       discover/search |      0.00347744  |      0.00347908  |       0       |  ops/s |   +0.05% |
|                                             Median Throughput |                                                       discover/search |      0.00353725  |      0.00353854  |       0       |  ops/s |   +0.04% |
|                                                Max Throughput |                                                       discover/search |      0.00455195  |      0.00455467  |       0       |  ops/s |   +0.06% |
|                                       50th percentile latency |                                                       discover/search |   3118.47        |   2992.23        |    -126.241   |     ms |   -4.05% |
|                                      100th percentile latency |                                                       discover/search |   3787.64        |   3878.36        |      90.7241  |     ms |   +2.40% |
|                                  50th percentile service time |                                                       discover/search |   1099.33        |   1390.95        |     291.622   |     ms |  +26.53% |
|                                 100th percentile service time |                                                       discover/search |   1774.22        |   1696.47        |     -77.7512  |     ms |   -4.38% |
|                                                    error rate |                                                       discover/search |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                               discover/search/no_minimized_roundtrips |      0.00205669  |      0.00205608  |      -0       |  ops/s |   -0.03% |
|                                               Mean Throughput |                               discover/search/no_minimized_roundtrips |      0.00249479  |      0.00249377  |      -0       |  ops/s |   -0.04% |
|                                             Median Throughput |                               discover/search/no_minimized_roundtrips |      0.00249479  |      0.00249377  |      -0       |  ops/s |   -0.04% |
|                                                Max Throughput |                               discover/search/no_minimized_roundtrips |      0.00293289  |      0.00293145  |      -0       |  ops/s |   -0.05% |
|                                       50th percentile latency |                               discover/search/no_minimized_roundtrips | 124688           | 125130           |     442.34    |     ms |   +0.35% |
|                                      100th percentile latency |                               discover/search/no_minimized_roundtrips | 149497           | 150018           |     521.141   |     ms |   +0.35% |
|                                  50th percentile service time |                               discover/search/no_minimized_roundtrips |  49845.1         |  50033.6         |     188.475   |     ms |   +0.38% |
|                                 100th percentile service time |                               discover/search/no_minimized_roundtrips |  50236.7         |  50453.5         |     216.754   |     ms |   +0.43% |
|                                                    error rate |                               discover/search/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                    discover/visualize |      0.00231261  |      0.00231337  |       0       |  ops/s |   +0.03% |
|                                               Mean Throughput |                                                    discover/visualize |      0.00485273  |      0.00485428  |       0       |  ops/s |   +0.03% |
|                                             Median Throughput |                                                    discover/visualize |      0.00517466  |      0.00517652  |       0       |  ops/s |   +0.04% |
|                                                Max Throughput |                                                    discover/visualize |      0.00683491  |      0.00683712  |       0       |  ops/s |   +0.03% |
|                                       50th percentile latency |                                                    discover/visualize |   1265.31        |   1221.07        |     -44.24    |     ms |   -3.50% |
|                                      100th percentile latency |                                                    discover/visualize |   2226.01        |   2160.46        |     -65.5498  |     ms |   -2.94% |
|                                  50th percentile service time |                                                    discover/visualize |   1104           |   1090.64        |     -13.3573  |     ms |   -1.21% |
|                                 100th percentile service time |                                                    discover/visualize |   2225.53        |   2141.38        |     -84.145   |     ms |   -3.78% |
|                                                    error rate |                                                    discover/visualize |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                            discover/visualize/no_minimized_roundtrips |      0.00218627  |      0.00218443  |      -0       |  ops/s |   -0.08% |
|                                               Mean Throughput |                            discover/visualize/no_minimized_roundtrips |      0.00307764  |      0.00307429  |      -0       |  ops/s |   -0.11% |
|                                             Median Throughput |                            discover/visualize/no_minimized_roundtrips |      0.003108    |      0.00310475  |      -0       |  ops/s |   -0.10% |
|                                                Max Throughput |                            discover/visualize/no_minimized_roundtrips |      0.00393865  |      0.0039337   |      -0       |  ops/s |   -0.13% |
|                                       50th percentile latency |                            discover/visualize/no_minimized_roundtrips | 123427           | 124351           |     923.664   |     ms |   +0.75% |
|                                      100th percentile latency |                            discover/visualize/no_minimized_roundtrips | 171022           | 172008           |     986.719   |     ms |   +0.58% |
|                                  50th percentile service time |                            discover/visualize/no_minimized_roundtrips |  50138.8         |  50366.7         |     227.906   |     ms |   +0.45% |
|                                 100th percentile service time |                            discover/visualize/no_minimized_roundtrips |  50323.3         |  50592.4         |     269.02    |     ms |   +0.53% |
|                                                    error rate |                            discover/visualize/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                                 kafka |      0.00501314  |      0.00501331  |       0       |  ops/s |    0.00% |
|                                               Mean Throughput |                                                                 kafka |      0.011064    |      0.0110643   |       0       |  ops/s |    0.00% |
|                                             Median Throughput |                                                                 kafka |      0.0108078   |      0.010808    |       0       |  ops/s |    0.00% |
|                                                Max Throughput |                                                                 kafka |      0.0183774   |      0.018378    |       0       |  ops/s |    0.00% |
|                                       50th percentile latency |                                                                 kafka |    579.42        |    570.366       |      -9.0545  |     ms |   -1.56% |
|                                      100th percentile latency |                                                                 kafka |    862.63        |    905.347       |      42.7164  |     ms |   +4.95% |
|                                  50th percentile service time |                                                                 kafka |    576.671       |    567.924       |      -8.74707 |     ms |   -1.52% |
|                                 100th percentile service time |                                                                 kafka |    788.854       |    681.839       |    -107.015   |     ms |  -13.57% |
|                                                    error rate |                                                                 kafka |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                         kafka/no_minimized_roundtrips |      0.00493846  |      0.00493647  |      -0       |  ops/s |   -0.04% |
|                                               Mean Throughput |                                         kafka/no_minimized_roundtrips |      0.0106933   |      0.0106898   |      -0       |  ops/s |   -0.03% |
|                                             Median Throughput |                                         kafka/no_minimized_roundtrips |      0.0105239   |      0.0105217   |      -0       |  ops/s |   -0.02% |
|                                                Max Throughput |                                         kafka/no_minimized_roundtrips |      0.0174484   |      0.017433    |      -2e-05   |  ops/s |   -0.09% |
|                                       50th percentile latency |                                         kafka/no_minimized_roundtrips |  15283.7         |  15296.3         |      12.6743  |     ms |   +0.08% |
|                                      100th percentile latency |                                         kafka/no_minimized_roundtrips |  27475.4         |  27779.1         |     303.639   |     ms |   +1.11% |
|                                  50th percentile service time |                                         kafka/no_minimized_roundtrips |   8917.96        |   8943.3         |      25.3374  |     ms |   +0.28% |
|                                 100th percentile service time |                                         kafka/no_minimized_roundtrips |   9144.62        |   9174.73        |      30.1025  |     ms |   +0.33% |
|                                                    error rate |                                         kafka/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                                 nginx |      0.00639487  |      0.00640231  |       1e-05   |  ops/s |   +0.12% |
|                                               Mean Throughput |                                                                 nginx |      0.0228754   |      0.0228882   |       1e-05   |  ops/s |   +0.06% |
|                                             Median Throughput |                                                                 nginx |      0.023571    |      0.0235786   |       1e-05   |  ops/s |   +0.03% |
|                                                Max Throughput |                                                                 nginx |      0.0338225   |      0.0338351   |       1e-05   |  ops/s |   +0.04% |
|                                       50th percentile latency |                                                                 nginx |   2734.4         |   2523.41        |    -210.982   |     ms |   -7.72% |
|                                       90th percentile latency |                                                                 nginx |   4648.48        |   4600.98        |     -47.4985  |     ms |   -1.02% |
|                                      100th percentile latency |                                                                 nginx |   5985.47        |   6053.42        |      67.9507  |     ms |   +1.14% |
|                                  50th percentile service time |                                                                 nginx |   2274.54        |   2298.54        |      24.0051  |     ms |   +1.06% |
|                                  90th percentile service time |                                                                 nginx |   3118.58        |   3059.32        |     -59.2573  |     ms |   -1.90% |
|                                 100th percentile service time |                                                                 nginx |   4316.53        |   3810.15        |    -506.381   |     ms |  -11.73% |
|                                                    error rate |                                                                 nginx |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                         nginx/no_minimized_roundtrips |      0.00554318  |      0.00554135  |      -0       |  ops/s |   -0.03% |
|                                               Mean Throughput |                                         nginx/no_minimized_roundtrips |      0.0133854   |      0.0133757   |      -1e-05   |  ops/s |   -0.07% |
|                                             Median Throughput |                                         nginx/no_minimized_roundtrips |      0.0147864   |      0.0147863   |      -0       |  ops/s |   -0.00% |
|                                                Max Throughput |                                         nginx/no_minimized_roundtrips |      0.0173465   |      0.0173397   |      -1e-05   |  ops/s |   -0.04% |
|                                       50th percentile latency |                                         nginx/no_minimized_roundtrips | 154031           | 154352           |     321       |     ms |   +0.21% |
|                                       90th percentile latency |                                         nginx/no_minimized_roundtrips | 297485           | 298397           |     912.525   |     ms |   +0.31% |
|                                      100th percentile latency |                                         nginx/no_minimized_roundtrips | 330457           | 331573           |    1116.44    |     ms |   +0.34% |
|                                  50th percentile service time |                                         nginx/no_minimized_roundtrips |  35692.6         |  35742.4         |      49.875   |     ms |   +0.14% |
|                                  90th percentile service time |                                         nginx/no_minimized_roundtrips |  52641.6         |  52898.8         |     257.241   |     ms |   +0.49% |
|                                 100th percentile service time |                                         nginx/no_minimized_roundtrips |  53357.6         |  53220.5         |    -137.156   |     ms |   -0.26% |
|                                                    error rate |                                         nginx/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                                apache |      0.0430125   |      0.0430118   |      -0       |  ops/s |   -0.00% |
|                                               Mean Throughput |                                                                apache |      0.0854283   |      0.0854202   |      -1e-05   |  ops/s |   -0.01% |
|                                             Median Throughput |                                                                apache |      0.0881506   |      0.0881434   |      -1e-05   |  ops/s |   -0.01% |
|                                                Max Throughput |                                                                apache |      0.135594    |      0.135573    |      -2e-05   |  ops/s |   -0.02% |
|                                       50th percentile latency |                                                                apache |    679.21        |    683.014       |       3.80383 |     ms |   +0.56% |
|                                       90th percentile latency |                                                                apache |    975.597       |    997.167       |      21.5702  |     ms |   +2.21% |
|                                      100th percentile latency |                                                                apache |   1319.49        |   1471.65        |     152.159   |     ms |  +11.53% |
|                                  50th percentile service time |                                                                apache |    643.467       |    625.495       |     -17.9714  |     ms |   -2.79% |
|                                  90th percentile service time |                                                                apache |    917.67        |    993.921       |      76.251   |     ms |   +8.31% |
|                                 100th percentile service time |                                                                apache |   1297.47        |   1469.62        |     172.153   |     ms |  +13.27% |
|                                                    error rate |                                                                apache |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                        apache/no_minimized_roundtrips |      0.0469826   |      0.0469767   |      -1e-05   |  ops/s |   -0.01% |
|                                               Mean Throughput |                                        apache/no_minimized_roundtrips |      0.0644584   |      0.0642278   |      -0.00023 |  ops/s |   -0.36% |
|                                             Median Throughput |                                        apache/no_minimized_roundtrips |      0.0676609   |      0.0673621   |      -0.0003  |  ops/s |   -0.44% |
|                                                Max Throughput |                                        apache/no_minimized_roundtrips |      0.0736745   |      0.0733709   |      -0.0003  |  ops/s |   -0.41% |
|                                       50th percentile latency |                                        apache/no_minimized_roundtrips | 105515           | 107784           |    2268.93    |     ms |   +2.15% |
|                                       90th percentile latency |                                        apache/no_minimized_roundtrips | 135162           | 136280           |    1118.7     |     ms |   +0.83% |
|                                      100th percentile latency |                                        apache/no_minimized_roundtrips | 150284           | 151696           |    1411.94    |     ms |   +0.94% |
|                                  50th percentile service time |                                        apache/no_minimized_roundtrips |   8864.11        |   8887.94        |      23.8359  |     ms |   +0.27% |
|                                  90th percentile service time |                                        apache/no_minimized_roundtrips |  13589.8         |  13802.8         |     213.025   |     ms |   +1.57% |
|                                 100th percentile service time |                                        apache/no_minimized_roundtrips |  53689.9         |  54078.8         |     388.863   |     ms |   +0.72% |
|                                                    error rate |                                        apache/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                           system/auth |      0.0618102   |      0.0618097   |      -0       |  ops/s |   -0.00% |
|                                               Mean Throughput |                                                           system/auth |      0.105605    |      0.105588    |      -2e-05   |  ops/s |   -0.02% |
|                                             Median Throughput |                                                           system/auth |      0.105841    |      0.105839    |      -0       |  ops/s |   -0.00% |
|                                                Max Throughput |                                                           system/auth |      0.135613    |      0.135519    |      -9e-05   |  ops/s |   -0.07% |
|                                       50th percentile latency |                                                           system/auth |    769.523       |    804.585       |      35.0623  |     ms |   +4.56% |
|                                       90th percentile latency |                                                           system/auth |   1451.1         |   1597.07        |     145.968   |     ms |  +10.06% |
|                                      100th percentile latency |                                                           system/auth |   2153.97        |   2854.41        |     700.449   |     ms |  +32.52% |
|                                  50th percentile service time |                                                           system/auth |    713.603       |    743.489       |      29.8864  |     ms |   +4.19% |
|                                  90th percentile service time |                                                           system/auth |   1418.39        |   1446.22        |      27.8297  |     ms |   +1.96% |
|                                 100th percentile service time |                                                           system/auth |   1714.06        |   1990.35        |     276.289   |     ms |  +16.12% |
|                                                    error rate |                                                           system/auth |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                   system/auth/no_minimized_roundtrips |      0.0310842   |      0.0309496   |      -0.00013 |  ops/s |   -0.43% |
|                                               Mean Throughput |                                   system/auth/no_minimized_roundtrips |      0.0430451   |      0.0429004   |      -0.00014 |  ops/s |   -0.34% |
|                                             Median Throughput |                                   system/auth/no_minimized_roundtrips |      0.0434394   |      0.0432975   |      -0.00014 |  ops/s |   -0.33% |
|                                                Max Throughput |                                   system/auth/no_minimized_roundtrips |      0.0519486   |      0.051781    |      -0.00017 |  ops/s |   -0.32% |
|                                       50th percentile latency |                                   system/auth/no_minimized_roundtrips | 399015           | 400762           |    1747.42    |     ms |   +0.44% |
|                                       90th percentile latency |                                   system/auth/no_minimized_roundtrips | 577365           | 580284           |    2918.41    |     ms |   +0.51% |
|                                      100th percentile latency |                                   system/auth/no_minimized_roundtrips | 590603           | 593199           |    2596.31    |     ms |   +0.44% |
|                                  50th percentile service time |                                   system/auth/no_minimized_roundtrips |   4974.38        |   4998.37        |      23.9854  |     ms |   +0.48% |
|                                  90th percentile service time |                                   system/auth/no_minimized_roundtrips |  49413.7         |  49469.8         |      56.1875  |     ms |   +0.11% |
|                                 100th percentile service time |                                   system/auth/no_minimized_roundtrips |  50250.8         |  50244           |      -6.84375 |     ms |   -0.01% |
|                                                    error rate |                                   system/auth/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                               system/syslog/dashboard |      0.0396284   |      0.0396286   |       0       |  ops/s |    0.00% |
|                                               Mean Throughput |                                               system/syslog/dashboard |      0.0542198   |      0.0545029   |       0.00028 |  ops/s |   +0.52% |
|                                             Median Throughput |                                               system/syslog/dashboard |      0.0544903   |      0.0545031   |       1e-05   |  ops/s |   +0.02% |
|                                                Max Throughput |                                               system/syslog/dashboard |      0.0667024   |      0.0667026   |       0       |  ops/s |    0.00% |
|                                       50th percentile latency |                                               system/syslog/dashboard |    576.769       |    588.487       |      11.7179  |     ms |   +2.03% |
|                                       90th percentile latency |                                               system/syslog/dashboard |    960.935       |    960.733       |      -0.20248 |     ms |   -0.02% |
|                                      100th percentile latency |                                               system/syslog/dashboard |   1280.93        |   1581.61        |     300.683   |     ms |  +23.47% |
|                                  50th percentile service time |                                               system/syslog/dashboard |    573.981       |    586.893       |      12.9116  |     ms |   +2.25% |
|                                  90th percentile service time |                                               system/syslog/dashboard |    642.932       |    652.93        |       9.99825 |     ms |   +1.56% |
|                                 100th percentile service time |                                               system/syslog/dashboard |    682.784       |    839.746       |     156.962   |     ms |  +22.99% |
|                                                    error rate |                                               system/syslog/dashboard |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                       system/syslog/dashboard/no_minimized_roundtrips |      0.0368127   |      0.0368037   |      -1e-05   |  ops/s |   -0.02% |
|                                               Mean Throughput |                       system/syslog/dashboard/no_minimized_roundtrips |      0.0527674   |      0.052687    |      -8e-05   |  ops/s |   -0.15% |
|                                             Median Throughput |                       system/syslog/dashboard/no_minimized_roundtrips |      0.0521717   |      0.0521559   |      -2e-05   |  ops/s |   -0.03% |
|                                                Max Throughput |                       system/syslog/dashboard/no_minimized_roundtrips |      0.0732733   |      0.0729109   |      -0.00036 |  ops/s |   -0.49% |
|                                       50th percentile latency |                       system/syslog/dashboard/no_minimized_roundtrips |  50852.7         |  51315.1         |     462.463   |     ms |   +0.91% |
|                                       90th percentile latency |                       system/syslog/dashboard/no_minimized_roundtrips |  83260.5         |  83913.6         |     653.087   |     ms |   +0.78% |
|                                      100th percentile latency |                       system/syslog/dashboard/no_minimized_roundtrips |  91742.3         |  91902.8         |     160.492   |     ms |   +0.17% |
|                                  50th percentile service time |                       system/syslog/dashboard/no_minimized_roundtrips |   8859.56        |   8885.84        |      26.272   |     ms |   +0.30% |
|                                  90th percentile service time |                       system/syslog/dashboard/no_minimized_roundtrips |   9029.46        |   9023.75        |      -5.70811 |     ms |   -0.06% |
|                                 100th percentile service time |                       system/syslog/dashboard/no_minimized_roundtrips |   9125.72        |   9285.02        |     159.297   |     ms |   +1.75% |
|                                                    error rate |                       system/syslog/dashboard/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                    system/syslog/lens |      0.0112183   |      0.0112195   |       0       |  ops/s |   +0.01% |
|                                               Mean Throughput |                                                    system/syslog/lens |      0.0186166   |      0.0186209   |       0       |  ops/s |   +0.02% |
|                                             Median Throughput |                                                    system/syslog/lens |      0.0157819   |      0.0157835   |       0       |  ops/s |   +0.01% |
|                                                Max Throughput |                                                    system/syslog/lens |      0.0478242   |      0.0478512   |       3e-05   |  ops/s |   +0.06% |
|                                       50th percentile latency |                                                    system/syslog/lens |    575.009       |    582.879       |       7.87042 |     ms |   +1.37% |
|                                       90th percentile latency |                                                    system/syslog/lens |    994.776       |   1026.55        |      31.7781  |     ms |   +3.19% |
|                                      100th percentile latency |                                                    system/syslog/lens |   1001.32        |   1079.7         |      78.3796  |     ms |   +7.83% |
|                                  50th percentile service time |                                                    system/syslog/lens |    573.311       |    581.091       |       7.78015 |     ms |   +1.36% |
|                                  90th percentile service time |                                                    system/syslog/lens |    992.783       |   1024.32        |      31.5378  |     ms |   +3.18% |
|                                 100th percentile service time |                                                    system/syslog/lens |    998.99        |   1077.58        |      78.5936  |     ms |   +7.87% |
|                                                    error rate |                                                    system/syslog/lens |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                            system/syslog/lens/no_minimized_roundtrips |      0.0111025   |      0.0111029   |       0       |  ops/s |    0.00% |
|                                               Mean Throughput |                            system/syslog/lens/no_minimized_roundtrips |      0.0208552   |      0.0208548   |      -0       |  ops/s |   -0.00% |
|                                             Median Throughput |                            system/syslog/lens/no_minimized_roundtrips |      0.0161463   |      0.0161468   |       0       |  ops/s |    0.00% |
|                                                Max Throughput |                            system/syslog/lens/no_minimized_roundtrips |      0.0488851   |      0.0488727   |      -1e-05   |  ops/s |   -0.03% |
|                                       50th percentile latency |                            system/syslog/lens/no_minimized_roundtrips |  13759.3         |  13795.5         |      36.1846  |     ms |   +0.26% |
|                                       90th percentile latency |                            system/syslog/lens/no_minimized_roundtrips |  17927.8         |  18090.7         |     162.849   |     ms |   +0.91% |
|                                      100th percentile latency |                            system/syslog/lens/no_minimized_roundtrips |  18775.8         |  19003           |     227.236   |     ms |   +1.21% |
|                                  50th percentile service time |                            system/syslog/lens/no_minimized_roundtrips |   8756.87        |   8766.46        |       9.5918  |     ms |   +0.11% |
|                                  90th percentile service time |                            system/syslog/lens/no_minimized_roundtrips |   9013.91        |   9038.98        |      25.0754  |     ms |   +0.28% |
|                                 100th percentile service time |                            system/syslog/lens/no_minimized_roundtrips |   9040.54        |   9078.23        |      37.6865  |     ms |   +0.42% |
|                                                    error rate |                            system/syslog/lens/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                       mysql/dashboard |      0.00224876  |      0.00224875  |      -0       |  ops/s |   -0.00% |
|                                               Mean Throughput |                                                       mysql/dashboard |      0.00864329  |      0.00864334  |       0       |  ops/s |    0.00% |
|                                             Median Throughput |                                                       mysql/dashboard |      0.00846034  |      0.00846033  |      -0       |  ops/s |   -0.00% |
|                                                Max Throughput |                                                       mysql/dashboard |      0.0148343   |      0.0148342   |      -0       |  ops/s |   -0.00% |
|                                       50th percentile latency |                                                       mysql/dashboard |    642.542       |    653.067       |      10.5246  |     ms |   +1.64% |
|                                       90th percentile latency |                                                       mysql/dashboard |   1170.79        |   1143.03        |     -27.7575  |     ms |   -2.37% |
|                                      100th percentile latency |                                                       mysql/dashboard |   1187.74        |   1159.72        |     -28.0231  |     ms |   -2.36% |
|                                  50th percentile service time |                                                       mysql/dashboard |    601.837       |    618.986       |      17.1494  |     ms |   +2.85% |
|                                  90th percentile service time |                                                       mysql/dashboard |    994.745       |   1081.43        |      86.6859  |     ms |   +8.71% |
|                                 100th percentile service time |                                                       mysql/dashboard |   1186.03        |   1138.47        |     -47.556   |     ms |   -4.01% |
|                                                    error rate |                                                       mysql/dashboard |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                               mysql/dashboard/no_minimized_roundtrips |      0.00220735  |      0.00220774  |       0       |  ops/s |   +0.02% |
|                                               Mean Throughput |                               mysql/dashboard/no_minimized_roundtrips |      0.00821147  |      0.00820956  |      -0       |  ops/s |   -0.02% |
|                                             Median Throughput |                               mysql/dashboard/no_minimized_roundtrips |      0.00824718  |      0.00824547  |      -0       |  ops/s |   -0.02% |
|                                                Max Throughput |                               mysql/dashboard/no_minimized_roundtrips |      0.0138337   |      0.0138294   |      -0       |  ops/s |   -0.03% |
|                                       50th percentile latency |                               mysql/dashboard/no_minimized_roundtrips |  41705.4         |  42098.7         |     393.279   |     ms |   +0.94% |
|                                       90th percentile latency |                               mysql/dashboard/no_minimized_roundtrips |  68860           |  69221.8         |     361.802   |     ms |   +0.53% |
|                                      100th percentile latency |                               mysql/dashboard/no_minimized_roundtrips |  71226.1         |  71594.7         |     368.523   |     ms |   +0.52% |
|                                  50th percentile service time |                               mysql/dashboard/no_minimized_roundtrips |   8784.74        |   8845.94        |      61.2022  |     ms |   +0.70% |
|                                  90th percentile service time |                               mysql/dashboard/no_minimized_roundtrips |   9188.74        |   9225.84        |      37.1081  |     ms |   +0.40% |
|                                 100th percentile service time |                               mysql/dashboard/no_minimized_roundtrips |   9928.44        |   9916.53        |     -11.9111  |     ms |   -0.12% |
|                                                    error rate |                               mysql/dashboard/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                                 redis |      0.00954254  |      0.0095418   |      -0       |  ops/s |   -0.01% |
|                                               Mean Throughput |                                                                 redis |      0.040223    |      0.039069    |      -0.00115 |  ops/s |   -2.87% |
|                                             Median Throughput |                                                                 redis |      0.0420558   |      0.040707    |      -0.00135 |  ops/s |   -3.21% |
|                                                Max Throughput |                                                                 redis |      0.0676705   |      0.0676668   |      -0       |  ops/s |   -0.01% |
|                                       50th percentile latency |                                                                 redis |    840.329       |    796.431       |     -43.898   |     ms |   -5.22% |
|                                       90th percentile latency |                                                                 redis |   1446.57        |   1546.12        |      99.5558  |     ms |   +6.88% |
|                                      100th percentile latency |                                                                 redis |   1618.49        |   1711.29        |      92.8014  |     ms |   +5.73% |
|                                  50th percentile service time |                                                                 redis |    816.534       |    784.275       |     -32.26    |     ms |   -3.95% |
|                                  90th percentile service time |                                                                 redis |   1411.01        |   1495.77        |      84.7587  |     ms |   +6.01% |
|                                 100th percentile service time |                                                                 redis |   1617.44        |   1710.43        |      92.9899  |     ms |   +5.75% |
|                                                    error rate |                                                                 redis |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                         redis/no_minimized_roundtrips |      0.00932912  |      0.00932597  |      -0       |  ops/s |   -0.03% |
|                                               Mean Throughput |                                         redis/no_minimized_roundtrips |      0.0247162   |      0.0246727   |      -4e-05   |  ops/s |   -0.18% |
|                                             Median Throughput |                                         redis/no_minimized_roundtrips |      0.0265476   |      0.0264838   |      -6e-05   |  ops/s |   -0.24% |
|                                                Max Throughput |                                         redis/no_minimized_roundtrips |      0.0337785   |      0.0337583   |      -2e-05   |  ops/s |   -0.06% |
|                                       50th percentile latency |                                         redis/no_minimized_roundtrips | 140943           | 142227           |    1283.94    |     ms |   +0.91% |
|                                       90th percentile latency |                                         redis/no_minimized_roundtrips | 416567           | 418330           |    1762.55    |     ms |   +0.42% |
|                                      100th percentile latency |                                         redis/no_minimized_roundtrips | 462866           | 464780           |    1913.88    |     ms |   +0.41% |
|                                  50th percentile service time |                                         redis/no_minimized_roundtrips |   9260.73        |   9340.36        |      79.6367  |     ms |   +0.86% |
|                                  90th percentile service time |                                         redis/no_minimized_roundtrips |  49804.1         |  50125           |     320.907   |     ms |   +0.64% |
|                                 100th percentile service time |                                         redis/no_minimized_roundtrips |  50371.7         |  50384.7         |      13.0156  |     ms |   +0.03% |
|                                                    error rate |                                         redis/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                            mysql/lens |      0.00675143  |      0.00675148  |       0       |  ops/s |    0.00% |
|                                               Mean Throughput |                                                            mysql/lens |      0.0272501   |      0.0272527   |       0       |  ops/s |    0.01% |
|                                             Median Throughput |                                                            mysql/lens |      0.028152    |      0.0281533   |       0       |  ops/s |    0.00% |
|                                                Max Throughput |                                                            mysql/lens |      0.0446939   |      0.044699    |       1e-05   |  ops/s |   +0.01% |
|                                       50th percentile latency |                                                            mysql/lens |    834.051       |    924.114       |      90.0637  |     ms |  +10.80% |
|                                       90th percentile latency |                                                            mysql/lens |   1321.22        |   1699.11        |     377.894   |     ms |  +28.60% |
|                                      100th percentile latency |                                                            mysql/lens |   1636.91        |   1814.86        |     177.943   |     ms |  +10.87% |
|                                  50th percentile service time |                                                            mysql/lens |    831.162       |    880.972       |      49.8101  |     ms |   +5.99% |
|                                  90th percentile service time |                                                            mysql/lens |   1184.03        |   1513.22        |     329.192   |     ms |  +27.80% |
|                                 100th percentile service time |                                                            mysql/lens |   1320.03        |   1813.85        |     493.819   |     ms |  +37.41% |
|                                                    error rate |                                                            mysql/lens |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                    mysql/lens/no_minimized_roundtrips |      0.00673111  |      0.00673078  |      -0       |  ops/s |   -0.00% |
|                                               Mean Throughput |                                    mysql/lens/no_minimized_roundtrips |      0.0208024   |      0.020778    |      -2e-05   |  ops/s |   -0.12% |
|                                             Median Throughput |                                    mysql/lens/no_minimized_roundtrips |      0.0216333   |      0.0216105   |      -2e-05   |  ops/s |   -0.11% |
|                                                Max Throughput |                                    mysql/lens/no_minimized_roundtrips |      0.025004    |      0.0250027   |      -0       |  ops/s |   -0.01% |
|                                       50th percentile latency |                                    mysql/lens/no_minimized_roundtrips | 103588           | 103805           |     216.727   |     ms |   +0.21% |
|                                       90th percentile latency |                                    mysql/lens/no_minimized_roundtrips | 227607           | 228834           |    1227.14    |     ms |   +0.54% |
|                                      100th percentile latency |                                    mysql/lens/no_minimized_roundtrips | 234501           | 235173           |     672.156   |     ms |   +0.29% |
|                                  50th percentile service time |                                    mysql/lens/no_minimized_roundtrips |  49329.4         |  49459.6         |     130.152   |     ms |   +0.26% |
|                                  90th percentile service time |                                    mysql/lens/no_minimized_roundtrips |  49831.8         |  50041.4         |     209.647   |     ms |   +0.42% |
|                                 100th percentile service time |                                    mysql/lens/no_minimized_roundtrips |  50063.9         |  50301.5         |     237.637   |     ms |   +0.47% |
|                                                    error rate |                                    mysql/lens/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                   postgresql/overview |      0.00512419  |      0.00512465  |       0       |  ops/s |    0.01% |
|                                               Mean Throughput |                                                   postgresql/overview |      0.0541027   |      0.0541052   |       0       |  ops/s |    0.00% |
|                                             Median Throughput |                                                   postgresql/overview |      0.0578911   |      0.0578943   |       0       |  ops/s |    0.01% |
|                                                Max Throughput |                                                   postgresql/overview |      0.0810353   |      0.0810387   |       0       |  ops/s |    0.00% |
|                                       50th percentile latency |                                                   postgresql/overview |   1209.5         |   1215.48        |       5.97931 |     ms |   +0.49% |
|                                       90th percentile latency |                                                   postgresql/overview |   1920.59        |   1866.42        |     -54.177   |     ms |   -2.82% |
|                                      100th percentile latency |                                                   postgresql/overview |   2555.11        |   2589.5         |      34.3867  |     ms |   +1.35% |
|                                  50th percentile service time |                                                   postgresql/overview |   1201.81        |   1206.74        |       4.92297 |     ms |   +0.41% |
|                                  90th percentile service time |                                                   postgresql/overview |   1528.56        |   1567.04        |      38.4787  |     ms |   +2.52% |
|                                 100th percentile service time |                                                   postgresql/overview |   2012.8         |   2106.23        |      93.4314  |     ms |   +4.64% |
|                                                    error rate |                                                   postgresql/overview |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                           postgresql/overview/no_minimized_roundtrips |      0.00504411  |      0.00504038  |      -0       |  ops/s |   -0.07% |
|                                               Mean Throughput |                           postgresql/overview/no_minimized_roundtrips |      0.0345048   |      0.0344581   |      -5e-05   |  ops/s |   -0.14% |
|                                             Median Throughput |                           postgresql/overview/no_minimized_roundtrips |      0.0365461   |      0.0364971   |      -5e-05   |  ops/s |   -0.13% |
|                                                Max Throughput |                           postgresql/overview/no_minimized_roundtrips |      0.0536954   |      0.0536251   |      -7e-05   |  ops/s |   -0.13% |
|                                       50th percentile latency |                           postgresql/overview/no_minimized_roundtrips | 164970           | 165752           |     782.703   |     ms |   +0.47% |
|                                       90th percentile latency |                           postgresql/overview/no_minimized_roundtrips | 226444           | 228138           |    1693.72    |     ms |   +0.75% |
|                                      100th percentile latency |                           postgresql/overview/no_minimized_roundtrips | 289993           | 291873           |    1880.34    |     ms |   +0.65% |
|                                  50th percentile service time |                           postgresql/overview/no_minimized_roundtrips |   6146.43        |   6172.06        |      25.6338  |     ms |   +0.42% |
|                                  90th percentile service time |                           postgresql/overview/no_minimized_roundtrips |  35582.7         |  35681.5         |      98.8133  |     ms |   +0.28% |
|                                 100th percentile service time |                           postgresql/overview/no_minimized_roundtrips |  36643.7         |  36545.1         |     -98.6133  |     ms |   -0.27% |
|                                                    error rate |                           postgresql/overview/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                   postgresql/duration |      0.0220715   |      0.0220712   |      -0       |  ops/s |   -0.00% |
|                                               Mean Throughput |                                                   postgresql/duration |      0.0680291   |      0.0678929   |      -0.00014 |  ops/s |   -0.20% |
|                                             Median Throughput |                                                   postgresql/duration |      0.065871    |      0.0658679   |      -0       |  ops/s |   -0.00% |
|                                                Max Throughput |                                                   postgresql/duration |      0.135703    |      0.135682    |      -2e-05   |  ops/s |   -0.02% |
|                                       50th percentile latency |                                                   postgresql/duration |    598.22        |    619.522       |      21.3022  |     ms |   +3.56% |
|                                       90th percentile latency |                                                   postgresql/duration |   1280.44        |   1360.09        |      79.6508  |     ms |   +6.22% |
|                                      100th percentile latency |                                                   postgresql/duration |   1600.27        |   1478.88        |    -121.392   |     ms |   -7.59% |
|                                  50th percentile service time |                                                   postgresql/duration |    526.065       |    543.846       |      17.7815  |     ms |   +3.38% |
|                                  90th percentile service time |                                                   postgresql/duration |    939.438       |   1202.5         |     263.061   |     ms |  +28.00% |
|                                 100th percentile service time |                                                   postgresql/duration |   1598.66        |   1380.72        |    -217.941   |     ms |  -13.63% |
|                                                    error rate |                                                   postgresql/duration |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                           postgresql/duration/no_minimized_roundtrips |      0.0220556   |      0.022057    |       0       |  ops/s |    0.01% |
|                                               Mean Throughput |                           postgresql/duration/no_minimized_roundtrips |      0.0519331   |      0.0517551   |      -0.00018 |  ops/s |   -0.34% |
|                                             Median Throughput |                           postgresql/duration/no_minimized_roundtrips |      0.0522277   |      0.0520396   |      -0.00019 |  ops/s |   -0.36% |
|                                                Max Throughput |                           postgresql/duration/no_minimized_roundtrips |      0.0750403   |      0.0749224   |      -0.00012 |  ops/s |   -0.16% |
|                                       50th percentile latency |                           postgresql/duration/no_minimized_roundtrips |  87629.3         |  88870           |    1240.75    |     ms |   +1.42% |
|                                       90th percentile latency |                           postgresql/duration/no_minimized_roundtrips | 287538           | 289331           |    1792.72    |     ms |   +0.62% |
|                                      100th percentile latency |                           postgresql/duration/no_minimized_roundtrips | 297789           | 300573           |    2783.66    |     ms |   +0.93% |
|                                  50th percentile service time |                           postgresql/duration/no_minimized_roundtrips |   4987.65        |   4934.97        |     -52.686   |     ms |   -1.06% |
|                                  90th percentile service time |                           postgresql/duration/no_minimized_roundtrips |  49897           |  50095.2         |     198.211   |     ms |   +0.40% |
|                                 100th percentile service time |                           postgresql/duration/no_minimized_roundtrips |  50407           |  50442.1         |      35.0273  |     ms |   +0.07% |
|                                                    error rate |                           postgresql/duration/no_minimized_roundtrips |      0           |      0           |       0       |      % |    0.00% |
|                                                Min Throughput |                                                            field-caps |      0.187197    |      0.187198    |       0       |  ops/s |    0.00% |
|                                               Mean Throughput |                                                            field-caps |      0.204705    |      0.204621    |      -8e-05   |  ops/s |   -0.04% |
|                                             Median Throughput |                                                            field-caps |      0.206076    |      0.205877    |      -0.0002  |  ops/s |   -0.10% |
|                                                Max Throughput |                                                            field-caps |      0.230659    |      0.230666    |       1e-05   |  ops/s |    0.00% |
|                                       50th percentile latency |                                                            field-caps |     95.9513      |     96.7132      |       0.76198 |     ms |   +0.79% |
|                                       90th percentile latency |                                                            field-caps |    118.829       |    102.555       |     -16.2741  |     ms |  -13.70% |
|                                       99th percentile latency |                                                            field-caps |    256.81        |    312.394       |      55.5838  |     ms |  +21.64% |
|                                      100th percentile latency |                                                            field-caps |    527.575       |    374.385       |    -153.19    |     ms |  -29.04% |
|                                  50th percentile service time |                                                            field-caps |     93.7208      |     94.4387      |       0.71795 |     ms |   +0.77% |
|                                  90th percentile service time |                                                            field-caps |    116.646       |     99.3151      |     -17.3314  |     ms |  -14.86% |
|                                  99th percentile service time |                                                            field-caps |    255.537       |    265.41        |       9.87331 |     ms |   +3.86% |
|                                 100th percentile service time |                                                            field-caps |    524.801       |    317.032       |    -207.768   |     ms |  -39.59% |
|                                                    error rate |                                                            field-caps |      0           |      0           |       0       |      % |    0.00% |


-------------------------------
[INFO] SUCCESS (took 0 seconds)
-------------------------------

```

</details>

Reply:
Addressed in https://github.com/elastic/rally/pull/1669/commits/407b5f5b7a5c54e2c53eea9be48671ae689cca71